conf/cassandra.yaml

```diff
@@ -675,6 +675,9 @@ replica_filtering_protection:
     # "cached_replica_rows_fail_threshold" is the per-query threshold at which the query will fail.
     #
     # These thresholds may also be adjusted at runtime using the StorageService mbean.
+    # 
+    # If the failure threshold is breeched, it is likely that either the current page/fetch size
+    # is too large or one or more replicas is severely out-of-sync and in need of repair.
```

Comment:
@adelapena If someone actually sees an error, it should be pretty easy for them to navigate back here and get a better understanding. WDYT?

Reply:
It looks perfect to me, +1